client/components/site-selector/style.scss

```diff
@@ -9,9 +9,6 @@
 	border: none;
 	z-index: z-index( 'root', '.site-selector' );
 
-	&.is-large .search  {
```

Comment:
Ups on that. Was trying to figure out how it works. So the Search is only visible when is-large is applied? If so I was just confused by the naming a bit.